### PR TITLE
[NO TICKET] Minor improvements in several areas

### PR DESCRIPTION
**Pull request by khairul-alam-licon**
# Description
This PR aims to include the following refactoring:
- Improve text of documentation for single dot(.) file downloading issue
- Move random contact generation to a common class e.g. `ContactHelper`
- Remove any "Deprecation" Suppress to visualize which implementation is deprecated 
- Change compress format from JPEG to PNG in `ProfileSettingsActivity`, similar to `SettingsFragment`
- Remove few unused files

# Checklist
- [x] I have read the [contributing guidelines](https://github.com/rakutentech/android-miniapp/blob/master/.github/CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](https://github.com/rakutentech/android-miniapp/blob/master/.github/CONTRIBUTING.md#sdk-development-learning-path) (if submitting a feature PR)
- [x] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

**Review by munir-rakuten — APPROVED** [review]
Looks good to me.

File: miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt [hunk 1]
```diff
@@ -41,8 +41,7 @@ import com.rakuten.tech.mobile.miniapp.storage.DownloadedManifestCache
     "ComplexMethod",
     "LongParameterList",
     " MaximumLineLength",
```
Comment (shohiebsense) [inline miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt:43]:
```suggestion
    "MaximumLineLength",
```

File: testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/userdata/ProfileSettingsActivity.kt [hunk 2]
```diff
@@ -107,7 +107,7 @@ class ProfileSettingsActivity : BaseActivity() {
                     inputStream?.use {
                         val bitmap = BitmapFactory.decodeStream(inputStream)
                         val byteArrayOutputStream = ByteArrayOutputStream()
-                        bitmap.compress(Bitmap.CompressFormat.JPEG, 50, byteArrayOutputStream)
+                        bitmap.compress(Bitmap.CompressFormat.PNG, 100, byteArrayOutputStream)
```
Comment (shohiebsense) [inline testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/userdata/ProfileSettingsActivity.kt:110]: what happened to JPEG? Just curious.

Comment (khairul-alam-licon) [reply testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/userdata/ProfileSettingsActivity.kt:110]: Yeah, we can keep either `JPEG` / `PNG`, so, I have just wanted to be the same with [SettingsFragment.kt#L350](https://github.com/rakutentech/android-miniapp/blob/master/testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/miniapptabs/fragments/SettingsFragment.kt#L350).

**Review by shohiebsense — APPROVED** [review]

**Pull request merged**